python-impl/ec.py

```diff
@@ -308,7 +307,7 @@ def bytes_to_point(buffer: bytes, ec, FE) -> JacobianPoint:
 
 def y_for_x(x, ec=default_ec, FE=Fq):
     """
-    Solves y = sqrt(x^3 + ax + b) for both valid ys
```

Comment:
This line says `Solves` and the above one `Check that`. 
I think a convention (active or passing) should be chosen. [Standard python](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) would probably be to do the latter (active job description, as in "Check that" and "Solve")